zk_toolbox/crates/zk_inception/src/commands/ecosystem/init.rs

```diff
@@ -358,3 +331,54 @@ async fn deploy_ecosystem_inner(
 
     Ok(contracts_config)
 }
+
+async fn init_chains(
+    init_args: &EcosystemInitArgs,
+    final_init_args: &EcosystemInitArgsFinal,
+    shell: &Shell,
+    ecosystem_config: &EcosystemConfig,
+) -> anyhow::Result<Vec<String>> {
+    // If the name of chain passed then we deploy exactly this chain otherwise deploy all chains
+    let list_of_chains = if let Some(name) = global_config().chain_name.clone() {
+        vec![name]
+    } else {
+        ecosystem_config.list_of_chains()
+    };
+    // Set default values for dev mode
+    let mut deploy_paymaster = init_args.deploy_paymaster;
+    let mut genesis_args = init_args.genesis_args.clone();
+    if final_init_args.dev {
+        deploy_paymaster = Some(true);
+        genesis_args.use_default = true;
```

Comment:
the initial decision, why we moved all these questions to the top, because we wanted to just ask all of them once and let user proceed. 
I'd vote to split this two process into two init ecosystem only and init all with chains